src/cli/index.ts

```diff
@@ -1,58 +1,71 @@
 import { globbySync } from "globby";
+import { parseArgs } from "./args-parser.js";
 import { UnsupportedExtensionError } from "../error/unsupported-extension-error.js";
 import { formatFile } from "../lib/format-file/index.js";
 import { LogUtils, LoggerVerboseOption } from "../utilities/log-utils.js";
+import { IgnoredFileError } from "../error/ignored-file-error.js";
+import { resolveOptions } from "../config/index.js";
 
 export function run(args: string[]) {
-  {
-    if (args.length === 0) {
+  const context = parseArgs(args);
```

Comment:
In general we need a ton more unit tests and to split this up a bit for better testing

Reply:
And now we're at 90% test coverage for the cli file!